nilearn/datasets.py

```diff
@@ -792,10 +792,10 @@ def fetch_yeo_2011_atlas(data_dir=None, url=None, resume=True, verbose=1):
     data: sklearn.datasets.base.Bunch
         dictionary-like object, keys are:
 
-        - "tight_7", "liberal_7": 7-region parcellations, resp. tightly
-          fitted to cortex shape, and liberally fitted.
+        - "thin_7", "thick_7": 7-region parcellations,
```

Comment:
I agree that these names are better but shouldn't we stick to the semantic used in the paper?